### PR TITLE
Feat: replaced all avatars with single custom avatar component, added tooltips

### PR DESCRIPTION
**Pull request by AndriiPopov3**

### CODE REVIEW

File: frontend/src/components/pages/components/response/response.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import Avatar from 'react-avatar';
 import { ListGroup } from 'react-bootstrap';
 import { getAllowedClasses } from 'helpers/dom/dom';
 import { replaceIdParam } from 'helpers/helpers';
 import { useHistory } from 'hooks/hooks';
 import { AppRoute } from 'common/enums/enums';
+import UserAvatar from '../../../common/avatar/avatar';
```
Comment (ursarik) [inline frontend/src/components/pages/components/response/response.tsx]:
```suggestion
import UserAvatar from 'common/avatar/avatar';
```

File: frontend/src/components/profile/components/profile-edit/profile-edit.tsx [hunk 2]
```diff
@@ -18,6 +17,7 @@ import { IUserAccount } from 'common/interfaces/user';
 import { useForm } from 'hooks/hooks';
 import { yupResolver } from 'hooks/hooks';
 import { accountInfoSchema } from 'validations/account-info-schema';
+import UserAvatar from '../../../common/avatar/avatar';
```
Comment (ursarik) [inline frontend/src/components/profile/components/profile-edit/profile-edit.tsx]:
```suggestion
import UserAvatar from 'common/avatar/avatar';
```

File: frontend/src/components/profile/components/profile-info/profile-info.tsx [hunk 3]
```diff
@@ -11,6 +10,7 @@ import { Link } from 'components/common/common';
 import './profile-info.scss';
 import { UserApi } from 'services';
 import { IUser } from 'common/interfaces/user';
+import UserAvatar from '../../../common/avatar/avatar';
```
Comment (ursarik) [inline frontend/src/components/profile/components/profile-info/profile-info.tsx]:
```suggestion
import UserAvatar from 'common/avatar/avatar';
```

**Review by ghost — APPROVED** [review]

File: frontend/src/components/workspace/team-settings/item/item.tsx [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import Avatar from 'react-avatar';
 import { Card } from 'react-bootstrap';
 import { ITeam, ITeamUser } from 'common/interfaces/team';
 import DropDown from '../drop-down/drop-down';
+import UserAvatar from '../../../common/avatar/avatar';
```
Comment (ursarik) [inline frontend/src/components/workspace/team-settings/item/item.tsx]:
```suggestion
import UserAvatar from 'common/avatar/avatar';
```

File: frontend/src/components/common/avatar/avatar.tsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    {({ ref, ...triggerHandler }): ReactElement => (
+      <div {...triggerHandler} ref={ref}>
+        <Avatar
+          size={size}
+          name={name}
+          src={src}
+          round={round}
+          className={className}
+          style={style}
+          onClick={onClick ? (): void => onClick() : undefined}
```
Comment (ursarik) [inline frontend/src/components/common/avatar/avatar.tsx]: why not
```suggestion
          onClick={onClick}
```
?

**Review by ursarik — APPROVED** [review]

**Pull request merged**